### ISSUE TITLE
docker

### ISSUE THREAD

**Issue by NavpreetSamra** [karesansui/karesansui#27]
is there any docker image available

**Comment by taizo** [comment]
No, I don't think so. It's quite difficult to make a docker image of karesansui because codes and documentations are outdated and making a image needs lots of effort for setup and installation.
